server/game/core/Player.js

```diff
@@ -636,16 +637,23 @@ class Player extends GameObject {
     getAdjustedCost(playingType, card, target, ignoreType = false) {
         // if any aspect penalties, check modifiers for them separately
         let aspectPenaltiesTotal = 0;
-        let aspects = card.aspects;
-        let cost = card.cost;
-
-        if (playingType === PlayType.Smuggle) {
-            const smuggleInstance = card.getKeyword(KeywordName.Smuggle);
-            if (smuggleInstance.hasCostValue()) {
+        let aspects;
+        let cost;
+
+        switch (playingType) {
+            case PlayType.PlayFromHand:
+                aspects = card.aspects;
+                cost = card.cost;
+                break;
+            case PlayType.Smuggle:
+                const smuggleInstance = card.getKeywordWithCostValues(KeywordName.Smuggle);
                 aspects = smuggleInstance.aspects;
                 cost = smuggleInstance.cost;
-            }
+                break;
+            default:
+                Contract.fail('Invalid Play Type');
```

Comment:
Ideally would be \`Invalid Play Type '${playingType}'\` so we get additional info in the log if we ever need to debug this